src/routes/conversation/[id]/+page.svelte

```diff
@@ -138,6 +138,7 @@
 			// eslint-disable-next-line no-undef
 			const encoder = new TextDecoderStream();
 			const reader = response?.body?.pipeThrough(encoder).getReader();
+			let importantUpdates: MessageUpdate[] = [];
```

Comment:
what;s the purpose of this variable `importantUpdates` ?

Reply:
I did it to solved the bug: when using Web Search and the llm finish outputing the web search box and the sources disappeared, and got visible again when refreshing the page.

the problem was that the updates of the current web search wasn’t saved into the updates in message inside the messages array, so I kept all the relevant updates (all without stream) in an array, and then saved it into the last message updates in the messages array.

Reply:
I will postpone the bug fix into a different PR, it isn’t related to this feature.